walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,100 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Iterator;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
 
+
+/*  Find time slot(s) that all the required attendees are availble
+    Include optional attendees if at least one time slot can be found
+    Otherwise return the time slots that fit just the mandatory attendees.
```

Comment:
nit: the documentation uses 'mandatory' and 'required' interchangeably. Not a big deal, but it's a good idea to stay consistent with terminology.